util/maven/generate_poms.py

```diff
@@ -58,6 +58,24 @@ def pom_deps(label):
         'name': 'Dagger',
         'artifact': 'dagger',
     },
+    '//java/dagger:gwt': {
+        'name': 'Dagger GWT',
+        'artifact': 'dagger-gwt',
+        'manual_dependencies': '''
```

Comment:
I like this solution better since the `:gwt` artifact isn't used anywhere else in the repo, so having `deps` to just get this working was funky. this is good :)